lib/oli/delivery/sections.ex

```diff
@@ -273,21 +273,44 @@ defmodule Oli.Delivery.Sections do
         e in Enrollment,
         join: s in Section,
         on: e.section_id == s.id,
-        where: s.slug == ^section_slug and s.status != :deleted,
+        where: s.slug == ^section_slug and s.status == :active,
         preload: [:user, :context_roles],
         select: e
       )
 
     Repo.all(query)
   end
 
+  @doc """
+  Returns a listing of all student enrollments for a given section.
+
+  """
+  def list_student_enrollments(section_slug) do
+    learner_role_id = ContextRoles.get_role(:context_learner).id
+
+    query =
+      from(
+        e in Enrollment,
+        join: s in Section,
```

Comment:
What do you think about `join: section in assoc(e, :section)` (saves 2 lines / easier to scan)

Reply:
Looks better :)